tools/go.mod

```diff
@@ -4,7 +4,7 @@ go 1.22
 
 require (
 	github.com/golang/mock v1.6.0
-	github.com/golangci/golangci-lint v1.55.2
+	github.com/golangci/golangci-lint v1.56.1
```

Comment:
need to run the linter manually on the branch to make sure lint warnings aren't hiding behind the only-new-issues: true setting isn't hiding them.